sec_edgar_downloader/_Downloader.py

```diff
@@ -173,6 +175,7 @@ def get(
                 download_details,
                 # Save ticker if passed in to form file system path for saving filings
                 ticker=ticker_or_cik if not is_cik(ticker_or_cik) else None,
+                skip_accession_numbers=skip_accession_numbers,
```

Comment:
This makes it seem like the value is a boolean, maybe rename to something like `accession_numbers_to_skip`